test/performance/odp_pool_perf.c

```diff
@@ -219,19 +239,34 @@ static int test_pool(void *arg)
 	c1 = odp_cpu_cycles();
 
 	for (rounds = 0; rounds < num_round; rounds++) {
-		num = odp_buffer_alloc_multi(pool, buf, max_burst);
+		num = 0;
+
+		for (i = 0; i < num_burst; i++) {
+			ret = odp_buffer_alloc_multi(pool, &buf[num],
+						     max_burst);
+			if (odp_unlikely(ret < 0)) {
+				printf("Error: Alloc failed. Round %u\n",
+				       rounds);
```

Comment:
Previously allocated buffers should be freed before exit.

Reply:
fixed